samples/wifi/provisioning/softap/README.rst

```diff
@@ -0,0 +1,51 @@
+.. _softap_wifi_provision_sample:
+
+Wi-Fi: SoftAP provision
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample demonstrates how to use the :ref:`lib_softap_wifi_provision` library to provision credentials to a Wi-Fi® device (nRF70 Series).
```

Comment:
Can we remove references to nRF70 Series? I believe the library and the sample are agnostic to the underlying hardware. I vote for Nordic Wi-Fi device, instead.

Reply:
Sure, I dont mind this. ^^ @divipillai

Reply:
Agree with the above comment. I have updated the sentence as : _This sample demonstrates how to use the :ref:`lib_softap_wifi_provision` library to provision credentials to Nordic Semiconductor's Wi-Fi® device._